packages/health-check/src/api/critical.js

```diff
@@ -0,0 +1,8 @@
+const db = require("../db");
+
+// returns all critical health check entries
+module.exports = (req, res) => {
+  const entries = db.get("critical").orderBy("date", "desc").value();
```

Comment:
Let's limit this to the last 24 hours. We don't really care about the health status from a month ago.

Also, let's purge old records from the DB. They won't take a lot of space but we don't need anything older than, say, a month.

Reply:
> Let's limit this to the last 24 hours. We don't really care about the health status from a month ago.

I will limit those to last 24 hours.

> Also, let's purge old records from the DB. They won't take a lot of space but we don't need anything older than, say, a month.

I'd like to keep them for now as is in case we set up a logger integrated with ELK stack that we could feed that information. Once we migrate archival data we will be able to add a purge mechanism.

Reply:
That makes sense.